### ISSUE TITLE
Method 'where' not found

### ISSUE THREAD

**Issue by M3hran** [lahell/PSDiscoveryProtocol#18]
is this the intended behavior ?

```
Method invocation failed because [Microsoft.Management.Infrastructure.CimInstance] does not contain a method named
'Where'.
At C:\Program Files\WindowsPowerShell\Modules\PSDiscoveryProtocol\1.4.0\PSDiscoveryProtocol.psm1:360 char:25
+ ...             $Adapter = (Get-NetAdapter -Physical).Where({ $_.Interfac ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Where:String) [], RuntimeException
    + FullyQualifiedErrorId : MethodNotFound

WARNING: No discovery protocol packets captured on WDS2019 in 32 seconds.
```

**Comment by lahell** [comment]
This is definitely not the intended behavior. Thank you for bringing it to my attention. In your example `Get-NetAdapter -Physical` returns a single network adapter. I did not catch this bug during testing because all of the computers returned multiple adapters. We should be able to fix it by changing `(Get-NetAdapter -Physical)` to `@(Get-NetAdapter -Physical)`.

**Comment by lahell** [comment]
Closed by commit 3776f5814a09b2ae90357868625330b386d86c1f.
